code/controllers/subsystem/ticker.dm

```diff
@@ -71,14 +71,19 @@ var/datum/subsystem/ticker/ticker
 		syndicate_code_response	= generate_code_phrase()
 	..()
 	start_at = world.time + (config.lobby_countdown * 10)
-	world << "<span class='boldnotice'>Welcome to [station_name()]!</span>"
 	world << "Please set up your character and select \"Ready\". The game will start in about [config.lobby_countdown] seconds."
-	current_state = GAME_STATE_PREGAME
 	for(var/client/C in clients)
 		window_flash(C, ignorepref = TRUE) //let them know lobby has opened up.
```

Comment:
also I just noticed you moved this here.

One thing to note, is alot of people will take about 10 to 15 seconds to reconnect after a round restart, more if byond decides they need to see an ad.

I put this with the GAME_STATE_STARTUP line to ensure more players saw it.

In fact i think it should be moved further down, maybe show with the tip?